src/adaption/Layers.cpp

```diff
@@ -42,7 +33,9 @@ Layers::Layers(FieldController *field_controller) {
 /// \brief  Checks for adaption
 /// \return  bool if adaption is possible true
 // ********************************************************************************
-bool Layers::update(long *p_shift_x1, long *p_shift_x2, long *p_shift_y1, long *p_shift_y2, long *p_shift_z1, long *p_shift_z2) {
+bool Layers::update(long *p_shift_x1, long *p_shift_x2,
+        long *p_shift_y1, long *p_shift_y2,
+        long *p_shift_z1, long *p_shift_z2) {
```

Comment:
according to the changes in the coding guidelines, it would be
```
bool Layers::update(
        long *p_shift_x1, ...
        long *p_shift_y1, ...
        long *p_shift_z1, ... )
```

Reply:
done in 62840f7